src/main/java/com/crud/tasks/service/DbService.java

```diff
@@ -3,21 +3,30 @@
 import com.crud.tasks.domain.Task;
 import com.crud.tasks.repository.TaskRepository;
 import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
-import java.util.Optional;
 
 @Service
 @RequiredArgsConstructor
 public class DbService {
+    @Autowired
     private final TaskRepository repository;
 
     public List<Task> getAllTasks() {
         return repository.findAll();
     }
 
-    public Optional<Task> getTask(final Long id) {
+    public Task getTask(final long id) {
         return repository.findById(id);
     }
+
+    public Task saveTask(final Task task) {
+         return repository.save(task);
+    }
+
+    public void deleteTask(Long id) {
+        repository.deleteById(id);
```

Comment:
a co jak chce wykasowac zadanie, ktorego nie ma? Moze najpierw trzeba sprawdzic, czy zadanie jest, a jesli nie ma, to rzucic wyjatek?